sound/soc/sof/sof-pci-dev.c

```diff
@@ -138,7 +138,7 @@ static const struct dmi_system_id community_key_platforms[] = {
 		.ident = "Google Chromebooks",
 		.callback = chromebook_use_community_key,
 		.matches = {
-			DMI_MATCH(DMI_SYS_VENDOR, "Google"),
```

Comment:
@jairaj-arava please remove the change ID

Reply:
> @jairaj-arava please remove the change ID

Done @ranj063 . Thanks